frontend/schema.js

```diff
@@ -47,6 +47,7 @@ export const DEFAULT_TIME_INTERVAL = 1209600;
 export const DEFAULT_TIME_INTERVAL_RELATIVE = 604800;
 export const DEFAULT_PERCENTILE = 99;
 export const DEFAULT_VERSION_GROUPING_TYPE = 'version';
+export const MAX_VISIBLE_SERIES = 5;
```

Comment:
After playing with this a bit, I think the right value of MAX_VISIBLE_SERIES depends quite a bit on the channel used. For release, 5 seems about right, but for beta and nightly the number should be lower. Could you turn this into an object with this structure:

```js
export const MAX_VISIBLE_SERIES = {
  'release': 5,
  'beta':  3,
  'nightly': 2
}
```

Then calculate the value in `detailview.jsx` like this:

```js
const maxVisibleSeries = MAX_VISIBLE_SERIES[this.state.channel];
```